### PR TITLE
Add some typescript definitions

### PR DESCRIPTION
**Pull request by tarrow**
This is the minimal set of TS definitions currently required by TaintedReferences

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+export class Claim {
+	getGuid(): string;
+    getMainSnak(): Snak;
```
Comment (mariushoch) [inline index.d.ts]: Spacing seems weird here

**Comment by mariushoch** [comment]
Do we have a way to verify these against the JS code (make sure that all things we map out here actually do exist)? If not, I think this is fine as is.

File: index.d.ts (hunk 1 above)
Comment (tarrow) [reply index.d.ts]: whoops, that'll teach me to change IDE halfway through

Comment (tarrow) [reply index.d.ts]: I think this is fixed now right

**Comment by Ladsgroup** [comment]
> I think that the repo is currently lacking webpack; 

It doesn't ^_^ look at the karma.conf.js

**Comment by Ladsgroup** [comment]
the test can be done later.

**Pull request merged**